packages/ckeditor5-dev-changelog/src/generatechangelog.ts

```diff
@@ -43,20 +44,36 @@ export async function generateChangelog( {
 	packagesDirectory = PACKAGES_DIRECTORY_NAME,
 	organisationNamespace = ORGANISATION_NAMESPACE,
 	externalRepositories = [],
+	// TODO: An integrator should define it. No defaults here.
+	// TODO: Required when `singlePackage=false`.
 	transformScope = defaultTransformScope,
 	date = format( new Date(), 'yyyy-MM-dd' ) as RawDateString,
 	changesetsDirectory = CHANGESET_DIRECTORY,
 	skipLinks = false,
 	singlePackage = false,
+
+	// TODO: Merge `removeInputFiles` and `noWrite` options.
 	noWrite = false,
 	removeInputFiles = true
 }: RepositoryConfig & GenerateChangelog ): Promise<string | void> { // eslint-disable-line @typescript-eslint/no-invalid-void-type
+	// TODO: getExternalRepositoriesWithDefaults => `normalizeRepositories`.
```

Comment:
I agree with all other TODOs, but I'm not sure about this one. Providing default values is different than normallization, and this util only provides default values.